### DISCUSSION TITLE
2022-01-18 Weekly Meeting 39 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#168] [Weekly Meetings]
Weekly Meeting 39 is happening on Wednesday January 18th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- working groups check in: 
  - [website group](https://github.com/omigroup/website)
  - [gLTF working group](https://github.com/omigroup/gltf-extensions)
  - [WASM working group](https://github.com/omigroup/omi-wasm-group)

### Announcements and Updates
- Website discussion consensus on platform. @antpb has built a proof of concept that you can get the link and credentials to access in the #omi-website chanel. The demo site there was built in WordPress. If there is no opposition to going with this platform Anthony's will grant folks access and his company is willing to donate the server hosting and management to OMI if helpful. This means next step would be finishing out the site content and pointing a domain to the IP Anthony provides.
- What does future structure look like Master discussion
  - New Roles in OMI discussion
- Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by Elirudite** [comment]
I'd like to get a consensus on what platform to use for our website.
I'd also like to put out an open call for any and all content submissions to put on the website. Looking for primarily some avatars from the community. These avatars would be used as imagery to further spruce up the design, for instance: 
![Echoparadigm Edit - Extended, w pics and core value cards](https://user-images.githubusercontent.com/10410478/150033036-bf613c43-2c9d-4513-833c-1e93484609e4.png)

But also open to any and all submissions (articles, projects, videos, audio, images, etc) to be linked through or uploaded to the website. 

All media can be submitted here: https://airtable.com/shrZAsV1MA5QJctYx

On another note, I'd like to propose that instead of electing new chairs, we instead take the list of chair responsibilities and break it up into discrete tasks that anybody from the community can assign themselves to do. This would solve the problem of unsustainable workload falling on the leaders, make it easier for new/old members to see exactly what they can do to get involved and contribute, as well as create more clear/open space for accountability + gratitude for the work people are doing here.
This can be a 3-month 'sprint' for us to test how well (or not) this method of organizing works.

**Comment by mrmetaverse** [comment]
- [1/12/21 Sync Agenda and Notes](https://github.com/omigroup/omigroup/discussions/161)
- [New Roles in OMI discussion](https://github.com/omigroup/omigroup/discussions/165)
- [What does future structure look like Master discussion](https://github.com/omigroup/omigroup/discussions/157)

**Comment by LightLodges** [comment]
There's interest in exploring best fit events for 2022 strategy such as hackathons - do we have interest in partnered hackathons with others and taking a leadership role in helping make that happen?

**Reply by madjin** [reply]
Here are notes from last discussion on hackathon planning (11-19-21): https://hackmd.io/@XR/web3xr-hackathons
I have an interest in helping with getting sponsors and designing a virtual world + we can use as a lobby for events and connecting resources / communities together: https://github.com/madjin/webxr-lobby
Been making open source templates folks can hack with also. Thinking closer to spring time is an appropriate goal for rolling this out.

**Comment by mrmetaverse** [comment]
Regarding async work and working groups. We have different groups, with di

https://hackmd.io/@mrmetaverse/omi-github-process

- OMI Github [Backlog Refinement](https://github.com/orgs/omigroup/projects/2#card-75916543): Meets the fourth Tuesday of every month at 22:05 - 22:50 UTC (contact Jesse @mrmetaverse)

**Reply by mrmetaverse** [reply]
We hope to craft a winning alignment strategy for aggregating tickets from each of our working groups backlogs (called Projects on github) to collate on our Holistic backlog. We hope to be able to sustain our workload in each working group, but rally around major victories, pain points, and blockers in each working group. We are one OMI, but we are made up of many types of contributors and divide work based on the different core objectives.

**Comment by mrmetaverse** [comment]
Yasan made a great point. In other countries many people think that the Metaverse is owned by Facebook. They asked if we have a communications working group, and suggested that we could spread the word about the metaverse and how it is not owned by one company. This is directly related to our efforts, and is a thing we discuss often as a group. :) Evo suggested "Let's set a time to make a collaborative story weavers session together, and create a regular cadence for the communications group to meet."

**Reply by mrmetaverse** [reply]
airtable link for sharing metaverse media: https://airtable.com/shrZAsV1MA5QJctYx

**Comment by mrmetaverse** [comment]
Q: How can we get up to date on gLTF to become contributors? 
- https://www.khronos.org/gltf/ 
  - Check out their blog posts
  - past talks on youtube
- In order to be helpful at the working group level, either be an expert in a specific domain (like an engine, or 3d modeling package), then there are also people who know a lot about the gLTF specification process itself. We want to ensure our work is accurate to the official gltf specification, and then also ensure that it is useful for real world applications.

**Reply by mrmetaverse** [reply]
>We're just getting to the stage of finalizing the audio emitter extension along with some implementations. If you're interested, come to your weekly glTF meetings, Thursdays at 2:30 PM PST.

**Comment by mrmetaverse** [comment]
WASM Group notes

**Reply by mrmetaverse** [reply]
WASM group might not be the best name. The Web assembly group is really focused on sandbox scripting for the Metaverse. How can user generated content contain behaviors that can be executed in the web or game engine? Like a modding API for the Metaverse.

Alternating weeks, they focus on standards and then alternating weeks they present demos and share knowledge.

**Reply by mrmetaverse** [reply]
next task: How will we scope our work?

**Comment by robertlong** [comment]
Notes from Rick W:
![image](https://user-images.githubusercontent.com/1753624/150188087-54d74288-b4ee-4f9c-9a57-ff17b2c0ea31.png)
